File: core/src/com/mygdx/pipopipette/block/HollowSquare.java
package com.mygdx.pipopipette.block;

import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.graphics.g2d.Sprite;
import com.badlogic.gdx.math.Rectangle;
import com.badlogic.gdx.utils.Array;

public class HollowSquare extends Block{
    //public Sprite sprite;
    public HollowBar[] neighbors;
    public HollowSquare(float x, float y) {
        super(x,y);
        dx=400;
        dy=400;
        rectangle = new Rectangle(x, y, dx, dy);
        isFree=false;

    }

    //public HollowBar[] FindNeighbors(Array<HollowBar> bar, int i) {
        //this.neighbors.add();
    //}

}

File: core/src/com/mygdx/xcube/block/HollowSquare.kt
package com.mygdx.xcube.block

import com.badlogic.gdx.math.Rectangle

class HollowSquare(x: Float, y: Float) : Block(x, y) {
    lateinit var neighbors: Array<HollowBar>
    var jsp = Testeur()

    init {
        dx = 400f
        dy = 400f
        rectangle = Rectangle(x, y, dx, dy)
        isFree = false
    }
//public HollowBar[] FindNeighbors(Array<HollowBar> bar, int i) {
    //    this.neighbors.add();
    //}
}